# SPDX-FileCopyrightText: Huawei Inc. 
# 
# SPDX-License-Identifier: Apache-2.0 
 
test/xts/acts/ActsFeatureAbilityTest: wait 8 seconds between each testcase

sometimes feature abilities fail to start because the previous testcase
still has't completely started or terminated its feature ability.
Increasing sleep time from 1 seconds to 8 seconds between each testcase.
 
Upstream-Status: Pending 
 
Signed-off-by: Francesco Pham <francesco.pham@huawei.com> 
diff --git a/aafwk/aafwk_standard/featureability/actsfeatureabilitytest/entry/src/main/js/test/FeatureAbilityJsunit.test.js b/aafwk/aafwk_standard/featureability/actsfeatureabilitytest/entry/src/main/js/test/FeatureAbilityJsunit.test.js
index 10e517f2c..1abaccd94 100644
--- a/aafwk/aafwk_standard/featureability/actsfeatureabilitytest/entry/src/main/js/test/FeatureAbilityJsunit.test.js
+++ b/aafwk/aafwk_standard/featureability/actsfeatureabilitytest/entry/src/main/js/test/FeatureAbilityJsunit.test.js
@@ -82,7 +82,7 @@ describe('ActsFeatureAbilityTest', function () {
         }
     }
     beforeEach(() => {
-        sleep(1000);
+        sleep(8000);
     })
 
     /**
